shared/charts/gpii-flowmanager/templates/deployment.yaml

```diff
@@ -37,3 +37,11 @@ spec:
         {{- end}}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
+        livenessProbe:
```

Comment:
Are we okay with default parameters for probes (initial delay, period, etc)? Should we make these parameters configurable on chart level?

Reply:
I'd leave as it is, and just add more code in the case that is needed. But as always, if I missed any point please let me know.